tooling/nargo/src/ops/execute.rs

```diff
@@ -228,15 +258,45 @@ pub fn execute_program_with_profiling<
     foreign_call_executor: &mut E,
 ) -> Result<(WitnessStack<F>, ProfilingSamples), NargoError<F>> {
     let profiling_active = true;
-    execute_program_inner(
+    match execute_program_inner(
         program,
         initial_witness,
         blackbox_solver,
         foreign_call_executor,
         profiling_active,
-    )
+    ) {
+        Ok((witness_stack, profiling_samples, ..)) => Ok((witness_stack, profiling_samples)),
```

Comment:
It seems `execute_program_inner` returns a tuple of two elements, so this `match` and mapping shouldn't be necessary.

Reply:
Oh, stuff changed I guess.